src/index.ts

```diff
@@ -2,9 +2,11 @@ import * as octokit from '@octokit/graphql-schema';
 import * as core from '@actions/core';
 import * as github from '@actions/github';
 import { GithubDiscussionClient } from "./GithubDiscussionClient";
-import { containsKeyword, containsNegativeReaction, containsPositiveReaction, hasNonInstructionsReply, exceedsDaysUntilStale, hasReplies } from './util';
+import { containsKeyword, containsNegativeReaction, containsPositiveReaction, exceedsDaysUntilStale, hasReplies, hasNonBotReply } from './util';
 import { DiscussionCommentEdge } from './generated/graphql';
 
+const PAGE_SIZE = parseInt(core.getInput('days-until-stale', { required: false })) || 50;
+const GITHUB_BOT = core.getInput('github-bot', { required: false}) || 'github-actions';
```

Comment:
/nit If we're only referencing the value in `util`, then we can just move this to `util` so it's not defined in two files

Reply:
> /nit If we're only referencing the value in `util`, then we can just move this to `util` so it's not defined in two files

the bot login might change in future and considering the future requirements which might involve using this bot name at different places, I chose to keep it as it is.